src/test/java/fr/xephi/authme/data/limbo/LimboServiceHelperTest.java

```diff
@@ -27,17 +29,17 @@
     public void shouldMergeLimboPlayers() {
         // given
         Location newLocation = mock(Location.class);
-        LimboPlayer newLimbo = new LimboPlayer(newLocation, false, "grp-new", false, 0.0f, 0.0f);
+        LimboPlayer newLimbo = new LimboPlayer(newLocation, false, Collections.singletonList("grp-new"), false, 0.0f, 0.0f);
         Location oldLocation = mock(Location.class);
-        LimboPlayer oldLimbo = new LimboPlayer(oldLocation, true, "grp-old", true, 0.1f, 0.8f);
+        LimboPlayer oldLimbo = new LimboPlayer(oldLocation, true, Collections.singletonList("grp-old"), true, 0.1f, 0.8f);
 
         // when
         LimboPlayer result = limboServiceHelper.merge(newLimbo, oldLimbo);
 
         // then
         assertThat(result.getLocation(), equalTo(oldLocation));
         assertThat(result.isOperator(), equalTo(true));
-        assertThat(result.getGroup(), equalTo("grp-old"));
+        assertThat(result.getGroups(), equalTo(Collections.singletonList("grp-new")));
```

Comment:
This should have rung an alarm bell — this test caught a change in behavior it was exactly meant to test ;)